scripts/griddle.jsx

```diff
@@ -345,16 +346,45 @@ var Griddle = React.createClass({
             //get the correct page size
             if(this.state.sortColumn !== "" || this.props.initialSort !== ""){
                 var sortProperty = _.where(this.props.columnMetadata, {columnName: this.state.sortColumn});
-                sortProperty = (sortProperty.length > 0 && sortProperty[0].hasOwnProperty("sortProperty") && sortProperty[0]["sortProperty"]) || null
+                //var test = sortProperty[0].hasOwnProperty("sortType");
+                //sortProperty = (sortProperty.length > 0 && sortProperty[0].hasOwnProperty("sortProperty") && sortProperty[0]["sortProperty"]) || null
+
+                //check if column passed in exists and is defined in columnMetadata
+                if(sortProperty.length > 0) {
+                  if(sortProperty[0].hasOwnProperty("sortType")) {
+                      if(sortProperty[0].sortType.toLowerCase() == "date") {
+                        //call new sort functions here
+                        data = SortHelper.sortDate(data,this.state.sortColumn,this.state.sortAscending);
+                      }
+                      else if(sortProperty[0].sortType.toLowerCase() == "money"){
+
+                      }
+                  }
+                  else {
+                    data = _.sortBy(data,this.state.sortColumn);
```

Comment:
This will work if its just the sortColumn but there are a couple other ways that the data can be sorted. Probably should leave the standard sort as 

```
data = _.sortBy(data, function(item){
  return sortProperty ? item[that.state.sortColumn||that.props.initialSort][sortProperty] :
    item[that.state.sortColumn||that.props.initialSort];
});
```

for now